addons/mail/static/src/components/link_preview_aside_view/link_preview_aside_view.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+    <t t-name="mail.LinkPreviewAsideView" owl="1">
+        <button class="o_LinkPreviewAside btn" t-attf-class="{{ className }}" t-on-click="linkPreviewAsideView.onClick" t-ref="root">
```

Comment:
```suggestion
        <button class="o_LinkPreviewAside btn" t-atf-class="className" t-on-click="linkPreviewAsideView.onClick" t-ref="root">
```
Why use `attf` if you're not going to use the formatting capabilities?

Reply:
This is JS template of a component in Discuss code. All of them have `t-attf-class="{{ className }}"`, so that they properly support the way of passing classnames between components in discuss code.

We use `t-attf-class` just because we had components that use `t-att-class` already. This is a quick fix in response to a breaking change from migrating OWL from v1 to v2 in saas-15.3.

So right now it's just formality to have this `t-attf-class`. We know we should improve code so it's not necessary to pass this, but it's not trivial to do properly, and it's a very low priority right now, so it's alright to do that way for the moment.